python/cog/server/http.py

```diff
@@ -115,12 +115,10 @@ def create_app(
 
         @app.get("/health-check")
         async def healthcheck_startup_failed() -> Any:
-            return jsonable_encoder(
-                {
-                    "status": app.state.health.name,
-                    "setup": attrs.asdict(app.state.setup_result),
-                }
+            setup = (
```

Comment:
this change not needed.  app.state.setup_result = result is always defined above.